src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx

```diff
@@ -0,0 +1,67 @@
+import { Checkbox, Flex, Toggle } from "@artsy/palette"
+import { sortBy } from "lodash"
+import React, { FC } from "react"
+import { get } from "v2/Utils/get"
+
+import { useArtworkFilterContext } from "../ArtworkFilterContext"
+import { OptionText } from "./OptionText"
+
+export const ArtistsFilter: FC = () => {
+  const { aggregations, ...filterContext } = useArtworkFilterContext()
+
+  const props = {
+    onSelect: value =>
+      filterContext.setFilter("includeArtworksByFollowedArtists", value),
+    selected: Boolean(
+      filterContext.currentlySelectedFilters()[
+        "includeArtworksByFollowedArtists"
+      ]
+    ),
+  }
+
+  const count = get(filterContext, f => f.counts.followedArtists, 0)
+
+  const items = aggregations.find(agg => agg.slice === "ARTIST")
+
+  if (!(items && items.counts)) {
+    return null
+  }
+
+  const toggleSelection = (selected, name) => {
+    let artistIDs = filterContext.currentlySelectedFilters().artistIDs.slice()
+    if (selected) {
+      artistIDs.push(name)
+    } else {
+      artistIDs = artistIDs.filter(item => item !== name)
+    }
+    filterContext.setFilter("artistIDs", artistIDs)
+  }
+
+  return (
+    <Toggle label="Artists" expanded>
+      <Flex flexDirection="column">
+        <Checkbox disabled={!count} {...props}>
+          <OptionText>Artists I Follow ({count})</OptionText>
+        </Checkbox>
+        {sortBy(items.counts, ["name"]).map(({ value, name }, index) => {
```

Comment:
Here is where the artists are sorted by their name and each checkbox is rendered.